recipes-qt/automotive/qtapplicationmanager_git.bbappend

```diff
@@ -2,4 +2,4 @@
 # provides "geniviextras" for Qt which QtAM uses for DLT-logging
 DEPENDS += "qtivi"
 
-SRCREV = "05a6ca178d7d95128a2e0053d06db56a844e4696"
+#SRCREV = "05a6ca178d7d95128a2e0053d06db56a844e4696"
```

Comment:
Ah damn, that should have been changed to the real srcrev instead, I will fix that.